usr/bin/nxdk-configure

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+. ${NXDK_DIR}/usr/share/nxdk/prelude.inl
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
```

Comment:
Not sure yet; had some trouble with autoconf scripts explicitly running stuff like "/usr/bin/sdl-config". Thanks.